FlowCryptAppTests/Functionallity/Services/Organisational Rules Service/ClientConfigurationServiceTest.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ClientConfigurationServiceTest.swift
+//  FlowCryptAppTests
+//
+//  Created by Anton Kharchevskyi on 10.09.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import XCTest
+@testable import FlowCrypt
+
+// check if Email Key Manager should be used test and other client configuration is consistent
+class ClientConfigurationServiceTest: XCTestCase {
+
+    var sut: ClientConfigurationService!
+    var organisationalRulesService = OrganisationalRulesServiceMock()
+
+    override func setUp() {
+        super.setUp()
+
+        sut = ClientConfigurationService(organisationalRulesService: organisationalRulesService)
+    }
+
+    func testCheckDoesNotUseEKM() {
+        // EKM should not be used if keyManagerUrl is nil
+        organisationalRulesService.clientConfiguration = ClientConfiguration(keyManagerUrl: nil)
+        XCTAssert(sut.checkShouldUseEKM() == .doesNotUseEKM)
+
+        // EKM should not be used if keyManagerUrl is nil
+        organisationalRulesService.clientConfiguration = ClientConfiguration(
+            flags: [],
+            keyManagerUrl: nil
+        )
```

Comment:
I think you wanted to add `XCTAssert(sut.checkShouldUseEKM() == .doesNotUseEKM)` here else this line has no effect

Reply:
thanks. fixed